codec-http/src/main/java/io/netty/handler/codec/http/websocketx/PingWebSocketFrame.java

```diff
@@ -56,12 +56,22 @@ public PingWebSocketFrame(boolean finalFragment, int rsv, ByteBuf binaryData) {
 
     @Override
     public PingWebSocketFrame copy() {
-        return new PingWebSocketFrame(isFinalFragment(), rsv(), content().copy());
+        return (PingWebSocketFrame) super.copy();
     }
 
     @Override
     public PingWebSocketFrame duplicate() {
-        return new PingWebSocketFrame(isFinalFragment(), rsv(), content().duplicate());
+        return (PingWebSocketFrame) super.duplicate();
+    }
+
+    @Override
+    public PingWebSocketFrame rduplicate() {
+        return (PingWebSocketFrame) super.rduplicate();
+    }
+
+    @Override
+    protected PingWebSocketFrame copy(ByteBuf content) {
```

Comment:
consider renaming `copy`. `copy` nomenclature is overloaded with `ByteBuf`'s copy.